tutorials/s4sdk-odata-service-cloud-foundry/s4sdk-odata-service-cloud-foundry.md

```diff
@@ -205,55 +212,60 @@ Any `ODataException` thrown by the OData call is caught and logged, before retur
 
 [ACCORDION-BEGIN [Step 5: ](Deploying the Project)]
 
-Depending on your chosen archetype and SAP Cloud Platform setup you can deploy the project on either `SAP Cloud Platform Neo` or `SAP Cloud Platform CloudFoundry`.
+Depending on your chosen archetype and SAP Cloud Platform setup you can deploy the project on either *SAP Cloud Platform Neo* or *SAP Cloud Platform Cloud Foundry*.
 
-![list of businesspartners from ERP](post-4-businesspartners-result.png)
+**Provide Destinations for your Application**
 
-You need to supply the destination of your SAP S/4HANA system before you deploy the new version to Cloud Foundry or run the application on the local server.
+In order for your application to run you need to provide it with information about your S/4HANA system. Your `BusinessPartnerService` needs an URL and a means of authentication in order to reach the system. This information is provided in the form of a destination which we retrieve (in a multi-tenant environment) by transparently accessing the destination service of SAP Cloud Platform. This is done by retrieving the destination via the `DestinationAccessor` by name, in our case `MyErpSystem`. There are multiple ways to create such a destination. For now we will provide it to our application by creating an environment variable that holds the information. We will create such a variable now both for a local deployment and a deployment on SCP Cloud Foundry.
 
 **Run on a Local Server**
-As mentioned in the Tutorial `Create a sample application on Cloud Foundry using SAP Cloud SDK` of this tutorial series, you can run the project on a local `TomEE` server. Here, you need to supply the destinations as an environment variable on your local machine (replace `set` with the corresponding commands to define environment variables on your command shell).
+
+As mentioned in the Tutorial `Create a sample application on Cloud Foundry using SAP Cloud SDK` of this tutorial series, you can run the project on a local `TomEE` server. Here, you need to supply the destinations as an environment variable on your local machine. How you set an environment variable depends on your OS. For Mac the command looks like this:
```

Comment:
We heed to give a Windows example as well (inline, not in the appendix).

Reply:
Updated